examples/check_integration_error.py

```diff
@@ -41,90 +45,87 @@
 clean_data["t2":0] *= 0.5
 fake_data_noise_std = 2.0
 clean_data.reorder(["ph1", "ph2", "vd"])
-bounds = (0, 200)  # seem reasonable to me
```

Comment:
I am going to use the bounds returned by the function so that the integrals are the same but we are truly testing the error as the name implies

Reply:
Not a good change